src/types/ledger/output_metadata.rs

```diff
@@ -31,6 +31,7 @@ pub struct OutputMetadata {
     pub transaction_id: TransactionId,
     pub booked: MilestoneIndexTimestamp,
     pub spent: Option<SpentMetadata>,
+    pub latest_milestone: Option<MilestoneIndexTimestamp>,
```

Comment:
Why is this field optional? There should always be a `latest_milestone` by design, right?